core/tests/ts-integration/tests/api/web3.test.ts

```diff
@@ -44,21 +44,36 @@ describe('web3 API compatibility tests', () => {
         const blockHash = (await alice.provider.getBlock(blockNumber)).hash!;
         const blockWithTxsByNumber = await alice.provider.getBlock(blockNumber, true);
         expect(blockWithTxsByNumber.gasLimit).toBeGreaterThan(0n);
-        let sumTxGasUsed = 0n;
 
+        // `ethers.Block` doesn't include `logsBloom` for some reason.
+        const blockWithTxsByNumberFull = await alice.provider.send('eth_getBlockByNumber', [blockNumberHex, true]);
```

Comment:
Nit: Is there any particular reason to fetch transactions (by setting the 2nd arg of the call to `true`) instead of hashes? Looks like only hashes are used.